### PR TITLE
Bump to version 0.8.8

### PR DESCRIPTION
**Pull request by max611**
Introduce the GRPC adapter

### CODE REVIEW

**Comment by thegedge** [comment]
I have low context on the gRPC adapter, but looking at the PR there was this request:

> To emphasize what `@byroot` said, this should likely be well tested before we merge into master and bump the version.

Have we tested this in any way?

**Comment by max611** [comment]
Ho sorry forgot to add that, we have #200 running on Theme Store and it's been error free for a week now. We've tested on the Theme Store Staging before that and raised error on purpose to test the adapter and everything ran smoothly.

**Comment by max611** [comment]
@thegedge to add on my previous comment, here's the PR on theme store: https://github.com/Shopify/shopify-theme-store/pull/3340

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# v0.8.7
```
Comment (jpittis) [inline CHANGELOG.md]: Wrong version.

Comment (max611) [reply CHANGELOG.md]: 🤦‍♂️

**Comment by max611** [comment]
To release the new version, here's what I usually do:

- Update CHANGELOG.md and version.rb.
- Merge the PR into master
- Draft a new Github release
- Deploy

**Comment by jpittis** [comment]
LGTM

**Pull request merged**